poppy/storage/cassandra/certificates.py

```diff
@@ -248,27 +248,41 @@ def get_certs_by_domain(self, domain_name, project_id=None,
             'domain_name': domain_name.lower(),
         }
 
-        query_string  = CQL_SEARCH_CERT_BY_DOMAIN
```

Comment:
I wonder how much of these additional params are actually necessary...  Can a domain be associated with multiple flavors, cert_types or projects?

If a domain exists in the `certificate_info` table, there are validators in place to prevent adding another of the same name.  In fact, domain_name is the primary key for the table.  

If that's the case -- this could be greatly simplified.

Reply:
Yes. There can only be a single record for any given domain name. Same domain name with other flavors/cert_types/projects will not be stored. Thats the reason only domain_name is part of the sql query.